rca/landingpages/utils.py

```diff
@@ -1,13 +1,17 @@
+from rca.utils.models import get_listing_image
+
+
 def news_teaser_formatter(item, image=None):
     item_as_dict = {}
 
     editorial_type = item.editorial_types.first()
     item_as_dict["type"] = editorial_type.type if editorial_type else ""
 
-    if image and item.hero_image:
-        item_as_dict["image"] = item.hero_image.get_rendition("fill-392x284").url
-        item_as_dict["image_alt"] = item.hero_image.alt
-    item_as_dict["formatted_date"] = item.published_at
+    listing_image = get_listing_image(item)
+    if image and listing_image:
```

Comment:
What is `image` supposed to be here?

Reply:
`image` is passed in optionally, if you want an image for the teaser, eg here https://github.com/torchbox/rca-wagtail-2019/blob/p3.2/sprint-2/rca/landingpages/models.py#L672 because only the one item here has an image, and there's no point sending images through that aren't going to be used

Reply:
Ah, so `image` is a boolean flag to indicate that an image is to be returned, rather than an image itself?

In that case, for readability I'd consider renaming `image` to something like `return_image` and making it default to `False` rather then `None`